### PR TITLE
Merge to main 2 3 22

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

**Review by VictoriaTouati-AVEVA — APPROVED** [review]
Changes look good!

**Review by VictoriaTouati-AVEVA — CHANGES_REQUESTED** [review]
It looks like lines 10, 12, 14, 16, 18 and 20 are bulleted? If so, let's get rid of the dashes.

File: content/overview/get-started/gs-asset-rules.md [hunk 1]
```diff
@@ -7,11 +7,17 @@ uid: gsAssetRules
 Use asset rules to automatically create assets based on streams you find by specifying a stream naming pattern. Asset rules work best if your stream names adhere to a consistent naming convention. For example, the following streams are similarly named, and each contains a timestamped series of measurements for similar pumps:
 
 - **MyData.NorthAmerica.SLTC.PumpA.Temperature**
+-
```
Comment (VictoriaTouati-AVEVA) [inline content/overview/get-started/gs-asset-rules.md:10]: If the blank lines have bullets, they need to be removed.

File: content/overview/get-started/gs-asset-rules.md [hunk 2]
```diff
@@ -30,7 +30,8 @@ To create a new asset rule, follow the steps in the these procedures.
 
 5. [Preview assets](#preview-assets-#5): Display a preview of the assets that will be created by this rule. 
 
-For more information about asset rules, see the following topics:
+For more information about asset rules, see [Asset rules](xref:ccAssetRules).
+
 
 - [Asset rules](xref:ccAssetRules)
```
Comment (VictoriaTouati-AVEVA) [inline content/overview/get-started/gs-asset-rules.md]: We've linked to this topic in line 33. There's no intro to the bullets that follow line 33.

File: content/overview/get-started/gs-asset-rules.md [hunk 3]
```diff
@@ -78,7 +78,8 @@ Assemble token values so that you can select the information you want to extract
 1. Finally, just **A** is selected. Use the option **letters until the end of stream name**, and name this token **Id**. 
 1. Select **Capture**.
 
-**Note**: Alternatively, you can bring in tokens by using the  **Stream Metadata** button. 
+**Note**: Alternatively, you can bring in tokens by using the **Stream Metadata** button. 
+
 
 1. In the left panel, select **Stream Metadata**. You see the two metadata tokens added to the stream by the metadata rule you just created, **Region** and **Site**. This can be a useful way to bring in more tokens for some streams. 
 2. Leave the **Region** and **Site** tokens unchecked.
```
Comment (VictoriaTouati-AVEVA) [inline content/overview/get-started/gs-asset-rules.md]: Renumber to "1." and insert space above

File: content/overview/get-started/gs-asset-rules.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/overview/get-started/gs-asset-rules.md]: took em out, good catch.

**Pull request merged**